### ISSUE TITLE
sam build --use-container doesn't remove the container

### ISSUE THREAD

**Issue by mojodna** [awslabs/aws-sam-cli#859]
### Description

`sam build --use-container` doesn't remove the container after building is complete.

### Steps to reproduce

```bash
sam build --use-container # on a Python project
docker ps -a | grep build-python
```

### Observed result

A container still exists:

```
980cc68b3f5a        lambci/lambda:python3.6                                                  "/var/lang/bin/pytho…"    2 minutes ago       Exited (0) 2 minutes ago                            eloquent_carson
```

### Expected result

No additional containers in `docker ps -a`. (Equivalent to `docker run --rm`.)

### Additional environment details

1. OS: macOS 10.14.1
2. `sam --version`: SAM CLI, version 0.9.0

**Comment by jfuss** [comment]
@mojodna Thanks for the report. Looks like we never call `self._container_manager.stop(container)` when `.run(container)` finishes [here](https://github.com/awslabs/aws-sam-cli/blob/develop/samcli/lib/build/app_builder.py#L248).

**Comment by jfuss** [comment]
Released in [v0.11.0](https://github.com/awslabs/aws-sam-cli/releases/tag/v0.11.0)

Closing
